### DISCUSSION TITLE
can not see the LogSize and Lag in the Consumers' Topic

### DISCUSSION THREAD

**Discussion by loxiaopohaiol** [sbt/sbt#6779] [Q&A]
My old kafka manager version is 1.3.3.14, and the kafka version is 0.11.0.0,  and now ,I upgrade my kafka manager to 3.0.0.2, I can see the consumer and topic list. but I can't see the LogSize and Lag in the Consumers' Topic. So how can I solve this problems?
![image](https://user-images.githubusercontent.com/48348273/148366963-ec7b6515-ddb2-486d-b974-932c06cd6b1a.png)

**Comment by eed3si9n** [comment]
How does this anything to do with sbt?

**Reply by eed3si9n** [reply]
I am going to delete this discussion later.
